templates/internet-of-things/contact-us-hannover.html

```diff
@@ -0,0 +1,222 @@
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="GCLID__c" id="GCLID__c" value="" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Facebook_Click_ID__c" id="Facebook_Click_ID__c" value="" />
+        </fieldset>
+      </form>
+    </div>
+  </div>
+</section>
+
+<script>
+  function createMessage() {
+    var message = "";
+    var formFields = document.querySelectorAll(".js-formfield");
+
+    formFields.forEach(function (formField) {
+      var comma = "";
+      var fieldTitle = formField.querySelector(".p-heading--5");
+      var inputs = formField.querySelectorAll("input");
+      if (fieldTitle) {
+      message += fieldTitle.innerText + "\r\n";
+      }
+      
+      inputs.forEach(function (input) {
+        switch (input.type) {
+          case "radio":
+            if (input.checked) {
+              message += comma + input.value + "\r\n\r\n";
+              comma = ", ";
+            }
+            break;
+          case "checkbox":
+            if (input.checked) {
+              var subSectionText = "";
+              var subSection = input
+                .closest('[class*="col-"]')
+                .querySelector(".js-sub-section");
+              if (subSection) {
+                subSectionText = subSection.innerText + ": ";
+              }
+
+              var label = formField.querySelector(
+                "span#" + input.getAttribute("aria-labelledby")
+              );
+
+              if (label) {
+                label = subSectionText + label.innerText;
+              } else {
+                label = input.getAttribute("aria-labelledby");
+              }
+              message += comma + label + "\r\n\r\n";
+              comma = ", ";
+            }
+            break;
+          case "text":
+            if (input.value !== "") {
+              message += comma + input.value + "\r\n\r\n";
+              comma = ", ";
+            }
+            break;
+          case "number":
+            if (input.value !== "") {
+              message += comma + input.value + "\r\n\r\n";
+              comma = ", ";
+            }
+            break;
+          case "textarea":
+            if (input.value !== "") {
+              message += comma + input.value + "\r\n\r\n";
+              comma = ", ";
+            }
+            break;
+        }
+      });
+    });
+
+    return message;
+  }
+
+  function handleSubmission(e) {
+    e.preventDefault();
+
+    var leadComments = document.getElementById("Comments_from_lead__c");
+    leadComments.value = createMessage();
+    
+    //Only submit Marketo forms
+    var forms = document.getElementsByTagName("form");
+    for (let i = 0; i < forms.length; i++) {
+      if (forms[i].action.includes("/marketo/submit")) forms[i].submit();
+    }
+  }
+  
+  (function init() {
+    var submitButton = document.querySelector(".p-button--positive");
+    submitButton.addEventListener("click", handleSubmission);
```

Comment:
1. You probably want to find the form and add an eventlistener to the submit event.